### PR TITLE
Ansible Integrations Test Playbooks

### PR DESCRIPTION
**Pull request by SergeBakharev**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Adds test playbooks for Ansible Integrations:
* Azure
* Linux
* Windows
* Cisco IOS
* Cisco NXOS
* Kubernetes
* VMware

Also fixes a minor documentation issue where the incorrect context path was provided in for "gather facts" style Ansible modules.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [x] No

To clarify, there are a few changes in the command context output definitions, but they are not breaking changes. The commands always returned output without "ansible_" in the context path, the definition was incorrect.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AnsibleMicrosoftWindows/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Ansible Microsoft Windows
+- Maintenance and stability enhancements.
+- Correct contextpath for win-chocolatey-facts command
```
Comment (yaakovpraisler) [inline Packs/AnsibleMicrosoftWindows/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Fixed contextPath outputs for the following command:
  - ***win-chocolatey-facts***
  - ***win-disk-facts***
  - ***win-product-facts***
```

File: Packs/AnsibleVMware/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Ansible VMware
+- Maintenance and stability enhancements.
+- Correct contextpath for vmware-host-facts command
```
Comment (yaakovpraisler) [inline Packs/AnsibleVMware/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Fixed contextPath outputs for the ***vmware-host-facts*** command.
```

**Review by yaakovpraisler — COMMENTED** [review]
looks great! most of my notes are docs notes.

File: Packs/AnsibleAlibabaCloud/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Ansible Alibaba Cloud
+- Maintenance and stability enhancements.
```
[the remark this answers is not in the document]
Comment (SergeBakharev) [reply Packs/AnsibleAlibabaCloud/ReleaseNotes/1_0_1.md:4]: That actually is automatically added by demisto-sdk when using the update release notes option. However I agree it isn't really correct. I have removed this line from all the notes.

File: Packs/AnsibleCiscoIOS/Integrations/AnsibleCiscoIOS/AnsibleCiscoIOS_description.md [hunk 4]
```diff
@@ -18,4 +18,4 @@ The user account used for initial SSH login access can be level 1, however the e
 
 ## Testing
 
-This integration does not support testing from the integration management screen. Instead it is recommended to use the `!IOS-facts`command providing an example `host` as the command argument. This command will connect to the specified network device with the configured credentials in the integration, and if successful output general information about the device.
+This integration does not support testing from the integration management screen. Instead it is recommended to use the `!IOS-facts`command providing an example `host` as the command argument. This command will connect to the specified network device with the configured credentials in the integration, and if successful output general information about the device.
```
[the remark this answers is not in the document]
Comment (SergeBakharev) [reply Packs/AnsibleCiscoIOS/Integrations/AnsibleCiscoIOS/AnsibleCiscoIOS_description.md:21]: demisto-sdk format must have made some sort of tweak. Good spot with the missing space after IOS-facts!

File: Packs/AnsibleMicrosoftWindows/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Ansible Microsoft Windows
+- Documentation and metadata improvements. 
+- Fixed contextPath outputs for the following command:
```
Comment (yaakovpraisler) [inline Packs/AnsibleMicrosoftWindows/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Fixed contextPath outputs for the following commands:
```

**Review by yaakovpraisler — APPROVED** [review]

**Comment by yaakovpraisler** [comment]
validation fails on skipped tests, integrations without tests and known backward compatibility breaks.

**Pull request merged**